packages/wallet/backend/migrations/20240215061440_add_active_column.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('transactions', async (table) => {
+    table.boolean('active').notNullable().defaultTo(true)
```

Comment:
replace with `deletedAt` column that will keep the time of deletion

Reply:
How can I detect the time of deletion of old transactions?

Reply:
use `updatedAt` from wallet address